rubrik_cdm/cluster.py

```diff
@@ -20,9 +20,118 @@
 
 
 class Cluster(Api):
-    """This class contains methods related to the managment of the Rubrik cluster itself.
+    """This class contains methods related to the management of the Rubrik cluster itself.
     """
 
+    def set_cluster_location(self, location,
```

Comment:
` def set_cluster_location(self, location, timeout=15)` should be a single line. If `pylint` is automatically moving into seperate lines you can add `# pylint: ignore` to the end of the line to prevent changes.